docs/src/userguide/migrating_to_cy30.rst

```diff
@@ -226,3 +226,16 @@ To make it easier to handle cases where your interpretation of type
 annotations differs from Cython's, Cython 3 now supports setting the
 ``annotation_typing`` :ref:`directive <compiler-directives>` on a
 per-class or per-function level.
+
+C++ postincrement/postdecrement operator
+========================================
+
+Cython 3 differentiates between pre/post-increment and pre/pos-decrement
+operators (Cython 0.29 implemented both as predecrement operator).
```

Comment:
```suggestion
operators (Cython 0.29 implemented both as pre(in/de)crement operator).
```

Not sure if this phrasing is clearer but it kind of implied that Cython 0.29 subtracted one whatever you did.